CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ set(CMAKE_CXX_STANDARD 14)
 file(STRINGS version.txt TORCHVISION_VERSION)
 
 option(WITH_CUDA "Enable CUDA support" OFF)
+option(USE_PYTHON "Link to Python when building" OFF)
```

Comment:
Should we keep the default value of `USE_PYTHON` ON to maintain the current behaviour? Thoughts?

Reply:
@fmassa suggested in https://github.com/pytorch/vision/issues/2692#issuecomment-1009783544 that the default behavior could be changed. I think it's fine to go safer and keep the old behavior if that's preferable.

Reply:
I see. I'll run the internal tests on this PR and if that goes without any errors, we can merge this.

Reply:
Internal tests: D33582385